api/handlers.go

```diff
@@ -1045,7 +1045,7 @@ func networkDisconnect(c *context, w http.ResponseWriter, r *http.Request) {
 	// Extract container info from r.Body copy
 	var disconnect apitypes.NetworkDisconnect
 	if err := json.NewDecoder(bodyCopy).Decode(&disconnect); err != nil {
-		httpError(w, fmt.Sprintf("Container is not specified"), http.StatusNotFound)
+		httpError(w, fmt.Sprint("Container is not specified"), http.StatusNotFound)
```

Comment:
Could you do that as @AkihiroSuda  said, and find the same issue with global search? @fate-grand-order